sys-fs/bees/bees-9999.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit multilib linux-info
+
+DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
+HOMEPAGE="https://github.com/Zygo/bees"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/kakra/bees.git"
+	EGIT_BRANCH="integration"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Zygo/bees/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+PATCHES=(
+	"${FILESDIR}/v0.5-gentoo_build.patch"
+)
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="tools test"
+
+RDEPEND=">=sys-apps/util-linux-2.30.2"
+DEPEND="$RDEPEND >=sys-fs/btrfs-progs-4.1"
```

Comment:
Please always use `${foo}`-style variables in ebuilds, and one dependency thingie per line. Also, aren't btrfs-progs needed at runtime as well?

Reply:
Hmm, now that you ask that... It used to be a pure build-time dependency. But actually, the service wrapper script uses it. Let me fix this.